src/views/Map/CustomerMap.js

```diff
@@ -0,0 +1,221 @@
+
+import React from 'react'
+import { AppSwitch } from '@coreui/react'
+import { Input, Row, FormGroup } from 'reactstrap';
+
+var map, infoWindow, marker, markers, geoCodeValue;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected variable declaration to be on a new line.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)